src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/DefaultLSPEditorService.cs

```diff
@@ -19,7 +18,7 @@
 using Microsoft.VisualStudio.Threading;
 using Task = System.Threading.Tasks.Task;
 
-namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+namespace Microsoft.VisualStudio.LanguageServer.ContainedLanguage
 {
     [Shared]
     [Export(typeof(LSPEditorService))]
```

Comment:
I'm pretty sure once we move off of our OnTypeFormatting hack this entire class may go away. Right @ajaybhargavb ?

Reply:
That's right

Reply:
Sounds good, let me know and I'll remove it. Or feel free to remove it yourself of course :) I have no dependency on it in our code. 

---
In reply to: [437621582](https://github.com/dotnet/aspnetcore-tooling/pull/1994#discussion_r437621582) [](ancestors = 437621582)